### PR TITLE
More and cheaper smartgun drums

### PR DESCRIPTION
**Pull request by BraveMole**
## About The Pull Request

SG drums are now only 5 points in req, and i double the amount available in marine vendors. And sg get one more drum in their essential kit

## Why It's Good For The Game

With the aim mode, SG are less critical than what they used to. They should not always beg for req for their expensive ammo. I've put the drums at the same price as miniguns one, wich seem way more fair to me.

## Changelog
:cl:
balance: SG drums price reduced to 5 points, 2 more available in each marine vendor and 1 more in their essential kit
/:cl:

### CODE REVIEW

**Comment by davidchan13** [comment]
Why not just give them a third drum from their own vendor?

**Comment by BraveMole** [comment]
> Why not just give them a third drum from their own vendor?

Okay, why not

**Comment by MLGTASTICa** [comment]
Good idea with extra  ammo from cargo , but more round-start ammo is bad , SG ammo is already extremly powerfull and will easy put a xeno into 50% sunder if  they sit infront of the gun.

**Comment by Tennessee116** [comment]
I thought smart guns were supposed to be removed with the introduction of aim-mode?

File: code/game/objects/machinery/vending/marine_vending.dm [hunk 1]
```diff
@@ -205,7 +205,7 @@
 		/obj/item/ammo_magazine/rifle/tx15_flechette = 50,
 		/obj/item/ammo_magazine/pistol/standard_pocketpistol = 50,
 		/obj/item/ammo_magazine/flamer_tank/large = 10,
-		/obj/item/ammo_magazine/standard_smartmachinegun = 2,
+		/obj/item/ammo_magazine/standard_smartmachinegun = 4,
```
Comment (TiviPlus) [inline code/game/objects/machinery/vending/marine_vending.dm:208]: Undo this

**Comment by neohenry** [comment]
> 
> 
> I thought smart guns were supposed to be removed with the introduction of aim-mode?

Not anymore. The PR was closed. https://github.com/tgstation/TerraGov-Marine-Corps/pull/5260

**Pull request merged**